ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/StreamedQueryResultImpl.java

```diff
@@ -127,6 +144,24 @@ private void handleErrorWhilePolling(final Throwable t) {
     log.error("Unexpected error while polling: " + t);
   }
 
+  @Override
+  public boolean hasContinuationToken() {
+    return this.continuationToken.isPresent() && this.continuationToken.get().get() != null;
+  }
+
+  @Override
+  public Optional<AtomicReference<String>> getContinuationToken() {
+    return this.continuationToken;
+  }
+
+  @Override
+  public CompletableFuture<StreamedQueryResult> retry() {
+    if (!this.hasContinuationToken()) {
```

Comment:
If we're going to throw an error with no continuation token, maybe it makes sense to call this `continue()`.

Reply:
`continue` is a reserved keyword, so i renamed it to `continueFromLastContinuationToken`